packages/upload-api/src/blob/get.js

```diff
@@ -0,0 +1,24 @@
+import * as Server from '@ucanto/server'
+import * as Blob from '@web3-storage/capabilities/blob'
+import * as API from '../types.js'
+import { BlobNotFound } from './lib.js'
+import { decode } from 'multiformats/hashes/digest'
+
+/**
+ * @param {API.BlobServiceContext} context
+ * @returns {API.ServiceMethod<API.BlobGet, API.BlobGetSuccess, API.BlobGetFailure>}
+ */
+export function blobGetProvider(context) {
+  return Server.provide(Blob.get, async ({ capability }) => {
+    const { digest } = capability.nb
+    if (!digest) {
+      return Server.fail('nb.digest must be set')
+    }
```

Comment:
the validator from the capability should already guarantee this validation. Did you have any typescript error? This should really not be needed

Reply:
I did not. I ended up adjusting the `store/add` code and that's why we got this check here. addressed!